dotNETStandard/HealthVault.Mobile/cs/Xamarin.Core/ViewModels/MedicationsMainViewModel.cs

```diff
@@ -98,7 +112,7 @@ private void UpdateDisplay(IEnumerable<Medication> items)
         {
             var medicationsMainPage = new MedicationsSummaryPage()
             {
-                BindingContext = new MedicationsSummaryViewModel(medication, this.ingredientChoices, this.thingClient, this.recordId, this.NavigationService),
+                BindingContext = new MedicationsSummaryViewModel(medication, this.connection, this.NavigationService),
```

Comment:
This is where IoC would actually be a good entry point. 
`DependencyService.Get<MedicationsSummaryViewModel>()`
or even better if we could use:
`DependencyService.Get<MedicationsSummaryPage>()`